Classes/GlobalStateExplorers/FLEXClassesTableViewController.m

```diff
@@ -78,8 +78,18 @@ - (void)updateTitle
 - (void)searchBar:(UISearchBar *)searchBar textDidChange:(NSString *)searchText
 {
     if ([searchText length] > 0) {
-        NSPredicate *searchPreidcate = [NSPredicate predicateWithFormat:@"SELF CONTAINS[cd] %@", searchText];
-        self.filteredClassNames = [self.classNames filteredArrayUsingPredicate:searchPreidcate];
+        NSArray *searchArray = [searchText componentsSeparatedByString: @" "];
+        int index = 0;  
```

Comment:
can you create a temp array `filteredClassNames` and use `filteredClassNames == nil` to replace this index?